paddle/majel/README.md

```diff
@@ -95,32 +95,170 @@ Because `variant` may be thought of as "multi-type, single value", we can utiliz
 
 ## implement Tensor in Paddle
 
+We aim to implement an independent tensor library, which provides flexiable and efficient tensor operations and can run on muti-devices(CPU and GPU). In brief, there are mainly two aspects to consider. First, we have to make encapsulation of the hardware for the tensor library, including device memory, computing handles, cuda streams and so on. Second, after the tensor libaray is able to access the hardware resources, we should make carefully design and abstraction of the upper levels and provide flexiable interfaces.  
```

Comment:
The computing handle  is a pointer  to an opaque structure holding the some CUDA library context, such as cublasHandle, cudnnHandle, cufftHandle